server/src/main/java/org/opensearch/search/aggregations/bucket/terms/StreamStringTermsAggregator.java

```diff
@@ -6,7 +6,7 @@
  * compatible open source license.
  */
 
-package org.opensearch.search.aggregations.bucket.terms;
```

Comment:
it seems like moving this class to a new package requires us to change access modifiers across different aggregation classes. If that's the case, I would just refrain from creating dedicated package for streaming